libafl_nyx/build.rs

```diff
@@ -0,0 +1,20 @@
+use std::process::Command;
+
+#[cfg(target_os = "linux")]
+fn main() {
+    println!("cargo:rerun-if-changed=build.rs");
+    // let output = Command::new("./build_nyx_support.sh").output().expect("can't run ./build_nyx_support.sh");
+    let status = Command::new("./build_nyx_support.sh")
```

Comment:
IIRC checking out git modules here can be problematic for crates.io, cc @andreafioraldi